### PR TITLE
README: Fix the missing parentheses

### PR DESCRIPTION
**Pull request by Evstrife**
Signed-off-by: Evstrife <wus217@wfu.edu>

Fix the missing parentheses on line 412

#daemondeacons

### CODE REVIEW

**Comment by awlauria** [comment]
ok to test

**Comment by awlauria** [comment]
@Evstrife can you please fix your commit message? 

Thanks!

**Comment by jsquyres** [comment]
> @Evstrife can you please fix your commit message?

Specifically, your commit message appears to be a bit jumbled:

```
j
    
Signed-off-by: Evstrife <wus217@wfu.edu>
    
Fix the missing parentheses on line 412
    
Signed-off-by: Siyu Wu wus217@wfu.edu
```

Can you have a good title line+content, followed by a single signed off line?

Thanks!

**Comment by awlauria** [comment]
@Evstrife you'll need to to do a `git commit --amend` and force push to change the message.

**Comment by Evstrife** [comment]
> Actually, looks like a change on line 407 snuck in. Can you fix that?

I changed it to 407 now. thx

File: README [hunk 1]
```diff
@@ -404,7 +404,7 @@ Compiler Notes
   compiler Open MPI was configured and compiled.
 
   There are up to three sets of Fortran MPI bindings that may be
-  provided depending on your Fortran compiler):
+  provided   (depending on your Fortran compiler):
```
Comment (jsquyres) [inline README:407]:
```suggestion
  provided (depending on your Fortran compiler):
```

Comment (Evstrife) [reply README:407]: fixed!

**Pull request merged**